### ISSUE TITLE
Tab selection lost when changing locale 

### ISSUE THREAD

**Issue by jmfrenzel** [samvera/hyrax#3766]
### Descriptive summary
QA Testing for Hyrax  
Hyrax Version: 3.0.0.pre.beta2  
Hyrax Branch: master  
Hyrax Revision: 05875cb8f40f4d0474f28562e117c009ca979645   

### Rationale
Changing the locale should keep the page/view content.
 
### Expected behavior
Changing locale keeps the pages/views content. 
 
### Actual behavior
Changing locale resets the page/view to the first (default tab).

### Steps to reproduce the behavior

1. go tohttps://nurax-dev.curationexperts.com/concern/file_sets/0z708w57j/edit?locale=en#permissions_display
2. change the language, e.g. "Deutsch"

### Related work
https://github.com/samvera/hyrax/pull/2864

**Comment by no-reply** [comment]
this is still an issue, and i regret having filed it into the backlog in the past.

i18n locale users shouldn't have to put up with confusing UI behavior.

**Comment by rjkati** [comment]
Tested on nurax-dev running Hyrax 4.0.0.beta1. This is still an issue.

**Comment by conorom** [comment]
Yup, it sure is. I was looking at this as one of my final tickets as the first Maintenance WG of this year was coming to a close.

As I was mulling over how exactly I would store active tabs so that they would survive a page reload I noticed that the Hyrax home page itself has _two_ sets of Bootstrap tabs on it, such that the URL fragment does not store the current state of active/visible tabs. This makes any solution much more awkward, and in fact sort of breaks good web design practice, I think (I'm no expert but it's nice if the URL indicates the state of the page).

![Uploading Screen Shot 2022-10-04 at 2.46.09 PM.png…]()

It's up for debate as to whether that should be addressed before this ticket.
It certainly gave me pause for thought on continuing with it as the sprint ended!

But if it's left as-is then I guess multiple active tabs need to be gathered/stored when the new locale link is being clicked (before the page load happens). Working in the console from the screen above:

```
active_bootstrap_tabs = []
[]

$('li.nav-item > a.nav-link.active').each(function() { active_bootstrap_tabs.push($(this).attr('href')); });
Ct.fn.init(2) [a#recentTab.nav-link.active, a.nav-link.active, prevObject: Ct.fn.init(1)]

active_bootstrap_tabs
(2) ['#recently_uploaded', '#tab-col2-second']
```

And then JS will force any tabs in this stored array tabs to the active state on page loads.

I won't be working on this now so I leave it up to someone else to weigh in on whether the multi-tab/URL problem should block this (i.e. a home page redesign, I guess?) or if it should proceed with a this issue as-is, using (perhaps) a solution as described here.
